.github/workflows/test-invoke-pip.yml

```diff
@@ -3,11 +3,17 @@ on:
   push:
     branches:
       - 'main'
+    paths-ignore:
+      - 'invokeai/frontend/**'
+      - 'docs/**'
   pull_request:
     types:
       - 'ready_for_review'
       - 'opened'
       - 'synchronize'
+    paths-ignore:
```

Comment:
How will a PR for Docs pass the Tests, when they won't trigger 🤔

Reply:
How can we skip this big test suite when a PR only includes changes to `docs/**` then? That's the goal, so changes to `docs/**` can merge quickly and without using our runners.

Reply:
I suggest that we should look into the new [merge queue](https://github.blog/changelog/2023-02-08-pull-request-merge-queue-public-beta/) feature asap, since it sounds like it would highly speed up our PR velocity, unecessary if docs or feature PR ^^